services/user/CommonApi/common/packages/common-lib/src/rpc.ts

```diff
@@ -58,6 +58,9 @@ export function siblingUrl(
 ): string {
     const currentUrl = new URL(baseUrl || window.location.href);
     const hostnameParts = currentUrl.hostname.split(".");
+    if (hostnameParts.join(".") === "psibase.localhost") {
+        baseUrlIncludesSibling = false;
+    }
```

Comment:
This seems unnecessary

Reply:
apparently. not sure what I tripped over initially that made this necessary.
Removed.

Reply:
ah, needed for Firefox (which doesn't handle locahost subdomains same as other browsers). I've added this to the *next* PR with informative comment.